Sources/MapboxNavigation/RouteMapViewController.swift

```diff
@@ -155,9 +155,8 @@ class RouteMapViewController: UIViewController {
         self.init()
         self.navigationService = navigationService
         self.delegate = delegate
-        automaticallyAdjustsScrollViewInsets = false
```

Comment:
Should we ticket out any tail work to ensure that the map view’s camera insets respect the safe area insets?

Reply:
This part was removed because since iOS 11 `automaticallyAdjustsScrollViewInsets` was deprecated. I'll check this in scope of navigation camera viewport API work.